src/movement.cpp

```diff
@@ -817,19 +817,19 @@ bool MoveEvent::executeStep(Creature* creature, Item* item, const Position& pos,
 	return scriptInterface->callFunction(4, true);
 }
 
-uint32_t MoveEvent::fireEquip(Player* player, Item* item, slots_t slot, bool boolean)
+uint32_t MoveEvent::fireEquip(Player* player, Item* item, slots_t slot, bool isCheck)
```

Comment:
While you're at it, can't you rename the variable to something more meaningful? Perhaps something that contains the keyword 'abilities'. 🤔